scripts/romanisim-make-stack

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+import datetime
+import logging
+import csv
+from copy import deepcopy
+import asdf
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from astropy.table import Table
+from astropy.io import fits
+from astropy.time import Time
+import galsim
+from romanisim import catalog, image, wcs, persistence
+from romanisim.parameters import default_parameters_dictionary
+from romanisim import log
+from romanisim import parameters
+
+NUMBER_OF_DETECTORS = 18
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Make a stack of demo images.',
+        epilog='EXAMPLE: %(prog)s mosaic_list.csv',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    # Command line Argument
+    parser.add_argument('input_file_name',
+                        type=str,
+                        metavar='mosaic_list.csv',
+                        # nargs=1,
+                        help=f'Input (csv) file containing lists of observation parameters:'
+                             f'ra, dec, roll_angle, optical_element, date, overhead_time, ms_table_number')
+
+    # Boresight
+    parser.add_argument('-b', '--boresight',
+                        action='store_true',
+                        default=False,
+                        help=('RA & Dec specifies location of boresight, not center of WFI.'))
+
+    # WCS Object Catalog
+    parser.add_argument('-c', '--catalog',
+                        type=str,
+                        metavar='object_table.csv',
+                        default=None,
+                        help='Object catalog file for wcs matching (csv)')
+
+    # Debug argument
+    parser.add_argument('-d', "--debug",
+                        action='store_true',
+                        help='Display verbose debug output')
+
+    # Level argument
+    parser.add_argument('-l', '--level',
+                        type=int,
+                        default=1,
+                        help='1 or 2, for L1 or L2 output')
+
+    # Create script making argument
+    parser.add_argument('-m', "--make_script",
+                        action='store_true',
+                        help='Create list of romanisim calls from csv file (e.g. for cluster usage).')
+
+    # Number of objects to simulate
+    parser.add_argument('-n', '--nobj',
+                        type=int,
+                        default=1000,
+                        help='Specify number of objects to simulate.')
```

Comment:
For this kind of case where we're making a lot of images, we probably don't want to generate an independent random set of sources on each image.  We want instead to generate a big catalog up front.  We should either force the users to supply this catalog or do a bit more with the catalog generation in the code.

Reply:
I moved this block to the beginning, and it centers on the midpoints of the RA & DEC range in the input file, with a radius spanning the range + FOV.